src/maintypes.jl

```diff
@@ -1,32 +1,35 @@
 """
-    AbstractManifold{F}
+    AbstractManifold{𝔽 <: AbstractNummbers}
```

Comment:
```suggestion
    AbstractManifold{𝔽}
```
This part is not in the type definition.